docs/general/serializers.md

```diff
@@ -135,13 +135,15 @@ With the `:json_api` adapter, the previous serializers would be rendered as:
 
 #### ::link
 
-e.g.
-
 ```ruby
-link :other, 'https://example.com/resource'
 link :self do
- href "https://example.com/link_author/#{object.id}"
+  href "https://example.com/link_author/#{object.id}"
 end
+link :author { link_author_url(object) }
+link :link_authors { link_authors_url }
+link :other, 'https://example.com/resource'
+link :posts { link_author_posts_url(object) }
+```
```

Comment:
somewhere we need to mention that the generic interface only supports `url_for`, the rails helpers also introduce e.g. `link_author_posts_url`

ok for followup pr